readthedocs/settings/docker_compose.py

```diff
@@ -110,14 +111,14 @@ def DATABASES(self):  # noqa
     AZURE_ACCOUNT_KEY = 'Eby8vdM02xNOcqFlqUwJPLlmEtlCDXJ1OUzFT50uSRZ6IFsuFq2UVErCz4I6tq/K1SZFPTOtr/KBHBeksoGMGw=='
     AZURE_CONTAINER = 'static'
     AZURE_STATIC_STORAGE_CONTAINER = AZURE_CONTAINER
-    AZURE_MEDIA_STORAGE_HOSTNAME = 'community.dev.readthedocs.io'
+    AZURE_MEDIA_STORAGE_HOSTNAME = PRODUCTION_DOMAIN
```

Comment:
In an ideal world, media and static would be different domains but Azurite had problems with CORS headers when I implemented this.

Reply:
It does look like it has since added support for CORS but I didn't spend the time to figure out how to add it. It isn't clear if it is allowed to set globally as opposed to per object either. If it's per object, that probably isn't useful to us.